crates/solver/src/solver/oneinch_solver.rs

```diff
@@ -114,6 +114,14 @@ impl OneInchSolver {
             // Use only 3 main route for cheaper trades.
             main_route_parts: Some(Amount::new(3).unwrap()),
             parts: Some(Amount::new(3).unwrap()),
+            dest_receiver: None,
+            referrer_address: None,
```

Comment:
For the other apis, we used our settlement contract address. But this can/should be done in another PR.